examples/python_django/README.md

```diff
@@ -19,17 +19,17 @@ $ sudo apt install -y python3.6 python3-pip
 - django-qr-code
 - didkit
 - Django
```

Comment:
should django be all lowercase?

Reply:
also, you mentioned, didkit appears twice :D

Reply:
Removed duplicate and updated ordering/capitalization on https://github.com/spruceid/didkit/pull/200/commits/a73aa46c1e7c5060a12a6046f8f16e9112f89baf.